src/main/java/org/sunbird/cb/hubservices/serviceimpl/ProfileService.java

```diff
@@ -64,65 +54,13 @@ public Response multiSearchProfiles(String userId, MultiSearch mSearchRequest, S
 		try {
 			List<String> connectionIdsToExclude = connectionService.findUserConnectionsV2(userId,
 					Constants.Status.APPROVED);
+			List<String> connectionIdsToExcludeForPending = connectionService.findUserConnectionsV2(userId,
+					Constants.Status.PENDING);
```

Comment:
can we use single neo4j query to find both Approved and Pending ??

Reply:
I checked no implementation for this